src/Jira/Api/Client/CurlClient.php

```diff
@@ -75,6 +75,10 @@ public function sendRequest(
 
 		if ( $method == 'GET' ) {
 			$url .= '?' . http_build_query($data);
+
+			if ( !is_array($data) ) {
+				throw new \InvalidArgumentException('Data must be an array.');
```

Comment:
Please mention in DocBlock via `@throws` tag, that this exception is being thrown.